app/src/main/java/com/example/parkingsystem/mvp/model/ReservationModel.java

```diff
@@ -1,29 +1,81 @@
 package com.example.parkingsystem.mvp.model;
 
 import com.example.parkingsystem.mvp.contract.ReservationContract;
+import com.example.parkingsystem.mvp.model.reservation.Reservation;
+import com.example.parkingsystem.mvp.model.reservation.ReservationInfomation;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
 
 public class ReservationModel implements ReservationContract.ReservationModelContract {
 
-    private String date;
-    private String time;
+    private static final String BUTTON_START_DATE = "Start date";
```

Comment:
I think that all these constants are using to show the toast at the end. this should be in a resource and be called from the view

Reply:
done